File: AtCoder Beginner Contest 196/B - Round Down.py
# https://atcoder.jp/contests/abc196/tasks/abc196_b

import math

X = input()

# A, B = map(int,input().split('.'))

L = list(X)

# math.floor(X)

if ('.' in L) == False:
    print(X)
else:
    A, B = map(int,X.split('.'))
    print(A)
# print(L)

File: AtCoder Beginner Contest 196/B - Round Down.cpp
// https://atcoder.jp/contests/abc196/tasks/abc196_b

#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main(){
  string s;
  cin >> s;
  
  string ss = "";

  for(int i = 0; i < s.length(); i++){
    if(s[i] == '.') break;
    ss += s[i];
  }

  cout << ss << endl;
  return 0;
}